internal/wasm/jit/jit_impl_amd64.go

```diff
@@ -3657,6 +3774,42 @@ func (c *amd64Compiler) compileMemoryFill() error {
 	return nil
 }
 
+func (c *amd64Compiler) compileTableInit(o *wazeroir.OperationTableInit) error {
```

Comment:
don't forget the boilerplate if any of this stuff is implementing an interface. if not, fine for no godoc, just seems like this is about an interface.